packages/cron/test/storage.spec.js

```diff
@@ -56,6 +56,11 @@ describe('cron - check user storage quotas', () => {
   ]
 
   beforeEach(async () => {
```

Comment:
A few thoughts on testing:
- It'd be nice to write tests to cover for the different scenarios on a more granular level. (ie. users that are restricted are not sent in the admin list)
- We should be testing the email client is called with the right inputs (stubbing it should be fine here)

Reply:
- Test added to exclude restricted users as part of [#1194](https://github.com/web3-storage/web3.storage/issues/1194)
- Added test using `sinon` to ensure email client is called with the correct parameters.

Reply:
Nice! I wonder if we could we write the test in a bit more declarative way. At the moment there's a lot of boilerplate to create the users, uploads etc and it hard to figure out what are the actual numbers per user etc.

Can it be simplified by having a function to create a user with a specific amount of bytes stored?
And define users/bytes as a array of objects?

Reply:
Added a helper method for the user with files data setup, easier to read in the `beforeAll` now